02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2029 @@
+{
```

Comment:
<p>Add in a bit here:</p><p> </p><p> </p><h2>Basic Hierarchy Properties </h2><p>Each hierarchy is an object of the <a href="https://docs.atoti.io/latest/api/atoti.Hierarchy.html#atoti.Hierarchy" target="_blank">atoti.Hierarchy</a> class. This means that we are able to interact with and manipulate hierarchies using their associated class properties and functions. </p><p><strong>Some of the properties allow us to understand how the hierarchies in the cube are structuresd, and this will really help us make more sensible decisions when it comes to data analysis. (this is the new bit)</strong></p><p>A few basic properties are as follows: </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Minor amendment, please, to keep it all in active voice, as per the brand guidelines :)</p><p> </p><p>Some of the properties allow us to understand how the structure of the hierarchies in the cube works, and this will really help us make more sensible decisions when it comes to data analysis.</p><p> </p><p>or</p><p> </p><p>Some of the properties allow us to understand the structure of the hierarchies in the cube, and this will really help us make more sensible decisions when it comes to data analysis.</p>

Reply:
<p>Hmm not sure if I like the wording of this:</p><p> </p><p><strong>"Some of the properties allow us to understand how the structure of the hierarchies in the cube works..."</strong></p><p> </p><p>It makes it seem as though knowing the properties is a dependency for understanding the structure of the hierarchies in the cube. I've amended this to the following as a reword:</p><p> </p><p><strong>"Using hierarchy properties give us additional insight into the specifics of each hierarchy in the cube, and help us make more sensible decisions when it comes to data analysis."</strong></p><p> </p><p>I feel like the amended version captured the intended connotation of the proposed amendment, but let me know otherwise!</p>

Reply:
Can we nix the "Using" in your counter-proposal?

Reply:
Fixed!